lib/nginx.py

```diff
@@ -45,7 +45,7 @@ def get_path_config():
     %s
 }
 """
-    root_template = """
+    root_template = r"""
```

Comment:
Changing a normal multi-line string to a raw string is a functional change. Why is this done along with updating the dependencies? And, what is the rationale for this change?

Reply:
no longer in mr.

Reply:
You resolved this conversation but it does appear to actually still be in the MR.